src/components/NoteItem/NoteItem.scss

```diff
@@ -47,3 +47,25 @@
         padding: 10px;
     }
 }
+
+.container-note--dark {
```

Comment:
Why do you copy such properties as paddins, cursor, transition etc. When you only need to change color?